tests/survey_demo_tests.js

```diff
@@ -0,0 +1,18 @@
+var assert = require('chai').assert;
+var sinon = require('sinon');
+require('../survey.js')
+
+suite('Demo test', function () {
+	setup(function() {
+		sinon.spy(SurveyMan.survey, 'getOptionById');
+	});
+
+	teardown(function() {
+		SurveyMan.survey.getOptionById.restore();
+	});
+
+	test('Demo test', function() {
+		SurveyMan.survey.getOptionById();
```

Comment:
@etosch , yeah, I am aware that the test is failing but I just want to know if the tests are working with mocha, chai, sinon. And please help me where I can find mock jsonobject,oid and all which are necessary for testing.

Thank you.